app/javascript/pages/QuestionNew.js

```diff
@@ -1,11 +1,66 @@
 import React, { Component } from 'react'
+import Box from '@mui/material/Box';
+import TextField from '@mui/material/TextField';
+import Button from '@mui/material/Button';
+import formpic from "../assets/questionform.jpeg";
+import { Redirect } from "react-router-dom";
+
+
 
 export default class QuestionNew extends Component {
+    constructor(props){
+        super(props);
+        this.state = {
+            form: {
+             post: "",
+             img_url: "",
+             user_id: this.props.current_user.id
+            },
+            submitted: false
+        }
+    }
+    handleChange = (name, e) => {
+        let { form } = this.state
+        form[name] = e.target.value
+        this.setState({form: form})
+    }
+    handleSubmit = () => {
+        this.props.createNewQuestion(this.state.form)
+        this.setState({submitted: true})
+      }
     render() {
         return (
-            <div>
-                This is the Question New Index
-            </div>
+            <Box id="form"
+            component="form"
+            sx={{ '&.MuiTextFeild-root': { m: 1, width: '25ch' },}}
```

Comment:
Is `Feild` a typo? Should it be `Field`?

Reply:
I copied and pasted from the MUI website... so this time, and only this time it wasn't my typo.